force-app/main/default/triggers/CaseTrigger.trigger

```diff
@@ -0,0 +1,17 @@
+trigger CaseTrigger on Case (before insert, before update) {
+
+    CaseTriggerHandler caseTriggerHandler = new CaseTriggerHandler();
+
+    if (Trigger.isBefore) {
+          
+        if (Trigger.isUpdate) {
```

Comment:
All this logic should be inside Trigger Handler. There should be nothing inside the trigger itself other than calling the Trigger Handler.